<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventas</class>
 <widget class="QMainWindow" name="Ventas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventas.ui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Ventas">
     <attribute name="title">
      <string>Ventas</string>
     </attribute>
     <widget class="QTableWidget" name="tb_Ventas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>781</width>
        <height>301</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>111</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. Factura</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cantidad</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cantidad:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>No. Factura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Guardar_Ventas">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Editar_Ventas">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Eliminar_Ventas">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Vaciar_Ventas">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Vaciar Campos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Ventas_Id">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Ventas_Cantidad">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Ventas_Nfactura">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Ventas_Descripcion">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
